cloudhub-2/modules/ROOT/pages/ps-create-configure-tgw.adoc

```diff
@@ -166,6 +166,51 @@ To test the connection, use the Network Tools application.
 
 For download and usage information about the Network Tools application, see https://help.mulesoft.com/s/article/How-To-Use-Network-Tools-Application[How To Use Network Tools Application^] (KB article).
 
+[[best-practices]]
+== Maintain the Transit Gateway 
+
+After you test the connection to your private space, periodically ensure your transit gateway connection is maintained. 
```

Comment:
```suggestion
After you test the connection to your private space, periodically maintain your transit gateway connection. 
```

Reply:
What does maintaining include?

Reply:
Should maintain be "check the connection to"?

Reply:
```suggestion
After you test the connection to your private space, periodically check the connection to your transit gateway. 
```